3차 멘토링을 위한 PR

LeeKyoWon opened the pull request:
<!-- 제목 : [FEAT] [BE] 구현한 기능 -->
<!-- #[이슈 번호] -->

## 📝Part
- [x] BE
- [ ] FE

  <br/>

## ✔️PR Type
- [x] 새로운 기능 추가
- [x] 버그 수정
- [x] 주요 코드 리펙토링 (성능 최적화 등)
- [ ] 문서 작업
- [ ] 기타 (기타 사항 기입)

  <br/>

## ✔️PR Checklist
- [ ] 커밋 메세지를 컨벤션에 맞게 잘 적용 하였나요?
- [ ] 테스트 코드 작성 / 단위 테스트 or 서비스 테스트 진행 하셨나요?

  <br/>

## 🔎 작업 내용
- 기능에서 어떤 부분이
- 구현되었는지 설명해주세요

jurio5 committed:
[refactor][FE] 즐겨찾기 페이지 네이게이션 바 추가

LeeKyoWon committed:
[Feat][FE] 호텔 미등록 사업자 페이지 생성

LeeKyoWon committed:
[Feat][BE] json 유틸 클래스에 String -> Map 메서드 추가

LeeKyoWon committed:
Merge pull request #121 from prgrms-be-devcourse/refactor-admin/business

[REFACTOR][FE][BE] Reponse 구조 및 디자인 변경

LeeKyoWon committed:
Merge pull request #122 from prgrms-be-devcourse/refactor-be-business

[REFACTOR][BE] 사업자 인증 방식 변경

universechoi committed:
[refactor] [BE] 사업자 등록시 역할 변경

universechoi committed:
[feat] [fe] 사업자 등록 후 메인페이지 이동 기능 추가

LeeKyoWon committed:
Merge pull request #123 from prgrms-be-devcourse/feature/booking-fe

[REFACTOR][BE][FE] 예약/결제 디자인 개선

LeeKyoWon committed:
Merge pull request #125 from prgrms-be-devcourse/refactor-be-admin

[FEAT] [BE][FE] 사업자 승인 시  Role 변경, 사업자 등록 시 메인페이지 이동 기능 추가

LeeKyoWon committed:
Merge pull request #124 from prgrms-be-devcourse/feature-member-front

[FEAT] [FE] 추가정보 이메일 정규 표현 식 변경 및 즐겨찾기 네비게이션 바 추가

LeeKyoWon committed:
Merge branch 'dev' of https://github.com/prgrms-be-devcourse/NBE3-4-2-Team01 into feature-review

LeeKyoWon committed:
Merge pull request #120 from prgrms-be-devcourse/fix/Hotel_availableHotels

[FIX] [BE] 예약 가능한 호텔 처리

LeeKyoWon committed:
Merge branch 'dev' of https://github.com/prgrms-be-devcourse/NBE3-4-2-Team01 into feature-review

LeeKyoWon committed:
Merge branch 'dev' of https://github.com/prgrms-be-devcourse/NBE3-4-2-Team01 into feature-review-front

LeeKyoWon committed:
[Feat][BE] 호텔 생성 시 Role Cookie 갱신

LeeKyoWon committed:
Merge pull request #126 from prgrms-be-devcourse/feature-review-front

[FEAT][FE] 미등록 사업자 페이지 생성

LeeKyoWon committed:
Merge pull request #127 from prgrms-be-devcourse/feature-review

[FEAT][BE] 호텔 생성 시 Role Cookie 갱신

LeeKyoWon committed:
[Feat][FE] 호텔 사업자 관리 페이지에는 즐겨찾기 버튼 제거

LeeKyoWon committed:
[Fix][FE] 호텔 등록 후 Role 쿠키 갱신 문제 해결

LeeKyoWon committed:
[Feat][FE] 회원가입 성공 시 역할에 맞는 페이지로 이동

LeeKyoWon committed:
[Refactor][BE] BaseInit 에 호텔 미등록 사업자 추가

sete3683 committed:
[FIX][FE] 예약 페이지 이동 안되는 문제 해결

LeeKyoWon committed:
[Feat][FE] 로그아웃 버튼 누르면 메인페이지로 이동

jurio5 committed:
[fix][BE] 엑세스 토큰 만료 시 리프레쉬 토큰을 체크하여 로그아웃 되도록 변경

LeeKyoWon committed:
Merge pull request #128 from prgrms-be-devcourse/feature-review-front

[Feat][BE][FE] BaseInit 수정 + 호텔 생성 후 문제 해결

LeeKyoWon committed:
Merge pull request #129 from prgrms-be-devcourse/feature/booking-fe

[FIX][FE] 예약 페이지 이동 안되는 문제 해결

LeeKyoWon committed:
Merge pull request #130 from prgrms-be-devcourse/dev

Dev -> Main 병합

jurio5 committed:
[fix][BE] 엑세스 토큰 만료 시 유효한 리프레쉬 토큰으로 재발급

universechoi committed:
[refactor] [BE] 사업자 등록시 국세청 api 자동 승인 적용

universechoi committed:
[fix] [BE] merge conflict 해결 - business 승인 방식 변경 반영

universechoi committed:
[feat] [FE] 사업자 등록 후 재 로그인 요구 추가

oneplast committed:
[FEAT] [BE] 유저 호텔 조회 시 검색 조건에 맞는 객실들만 출력

oneplast committed:
[REFACTOR] [FE] 잔여 객실에 따른 예약 및 객실 상세 페이지 진입 버튼 구현

oneplast committed:
[FIX] [BE] [FE] 기능 시험 중 버그 수정

LeeKyoWon committed:
Merge pull request #131 from prgrms-be-devcourse/feature-member

[FEAT] [BE] 액세스 토큰 재발급 및 로그아웃

LeeKyoWon committed:
Merge pull request #132 from prgrms-be-devcourse/refactor-be-businessApi

[REFACTOR] [BE] 사업자 승인 방식 재변경 (국세청 API 사용)

LeeKyoWon committed:
Merge pull request #133 from prgrms-be-devcourse/feature/RoomList_available

[FEAT] [BE] 유저 호텔 조회 시 검색 조건에 맞는 객실들만 출력하는 로직 생성

LeeKyoWon committed:
Merge pull request #134 from prgrms-be-devcourse/feature/FE_RoomList_available

[REFACTOR] [FE] 잔여 객실에 따른 예약 및 객실 상세 페이지 진입 버튼 구현

LeeKyoWon committed:
Merge pull request #135 from prgrms-be-devcourse/dev

Dev -> Main 병합용 PR

LeeKyoWon committed:
[Refactor][FE] 호텔 상세 페이지의 하트 모양은 손님에게만 표시

LeeKyoWon committed:
[Refactor][FE] 서울시 구를 가나다 순으로 정리

LeeKyoWon committed:
[Design][FE] 메인페이지 디자인 수정

LeeKyoWon committed:
Merge pull request #136 from prgrms-be-devcourse/feature-review-front

[Refactor][[FE] 메인페이지, 서울시 구 enum, 호텔 상세페이지 좋아요 버튼 리팩토링

LeeKyoWon committed:
Merge pull request #137 from prgrms-be-devcourse/dev

Dev -> Main 병합

universechoi committed:
[feat] [be] 사용 중인 옵션에 대한 삭제 방지 및 예외 처리 추가

LeeKyoWon committed:
[Refactor][FE] 호텔 목록 페이지에서 호텔 상세 페이지로 personal 전달

universechoi committed:
[feat] [fe] 사용 중인 옵션 삭제 및 삭제시 오류 메세지 alert 추가

LeeKyoWon committed:
[Fix][FE] 호텔 상세 페이지 조회 시, 백엔드로 데이터 전송 안하는 문제 해결

LeeKyoWon committed:
Merge pull request #138 from prgrms-be-devcourse/feat-options-delete

[FEAT] 옵션 삭제 및 사용중인 옵션 삭제 시 오류메세지 alert 기능 추가

universechoi committed:
Merge pull request #139 from prgrms-be-devcourse/feature-review-front

[FIX][FE] 호텔 상세페이지에서 인원 수 필터링 안되는 문제 해결

sete3683 committed:
[FEAT][FE] 예약,결제 관련 기능 개선

- 배경색 추가
- 매출 조회 버그 수정
- 완료된 예약에 한해서만 리뷰 작성 가능하도록 변경
- 디자인 소폭 수정

LeeKyoWon committed:
[Feat][FE] 페이징 처리 적용

LeeKyoWon committed:
Merge pull request #140 from prgrms-be-devcourse/feature/booking-fe

[FEAT][FE] 예약,결제 관련 기능 개선

oneplast committed:
[fix] 프론트엔드 코드 커밋 제거

LeeKyoWon committed:
Merge pull request #141 from prgrms-be-devcourse/fix/Hotel_search

[FIX] [BE] [FE] 검색 버그 수정

LeeKyoWon committed:
Merge pull request #142 from prgrms-be-devcourse/feature-review-front

[Feat][FE] 페이징 처리 적용

LeeKyoWon committed:
[Feat][FE] 페이징 처리 오류 해결

LeeKyoWon committed:
Merge pull request #143 from prgrms-be-devcourse/feature-review-front

[Feat][FE] 페이징 처리 오류 해결

oneplast committed:
[REFACTOR] [FE] 객실 이름 폰트 파란색으로 교체

LeeKyoWon committed:
[Feat][FE] 페이징 에러 최종 해결

Account-4-Review commented on backend/src/main/java/com/ll/hotel/domain/booking/booking/controller/BookingController.java:
```diff
@@ -21,136 +18,88 @@ public class BookingController {
-                "예약 및 결제에 성공하였습니다.",
-                BookingResponse.from(booking)
+                "200",
+                "예약 페이지 정보 조회에 성공했습니다.",
```
의미 없는 데이터라고 생각해요.
에러 메시지가 아니면 아예 저걸 제거했으면 합니다.
단순히 삭제만 제거하는게 아니라, 의미 있는 데이터가 아니라면 전부 제거해 주세요.

Account-4-Review commented on backend/src/main/java/com/ll/hotel/domain/booking/booking/service/BookingService.java:
```diff
@@ -1,119 +1,209 @@
-            Optional<Room> room = roomRepository.findById(bookingRequest.roomId());
-            Optional<Hotel> hotel = hotelRepository.findById(bookingRequest.hotelId());
+            Room room = roomRepository.findById(bookingRequest.roomId())
+                    .orElseThrow(() -> new ServiceException("404", "객실 정보를 찾을 수 없습니다."));
```
예외를 일일히 텍스트로 쓰는 것 보단, 별도의 Enum 으로 담아서 공통화를 하는게 좋을 것 같아요.
특히나 어떤 정보를 찾을 수 없다 <- 이런건 다른데에서도 많이 쓰일텐데, 그때마다 일일히 만들면 그것만큼 심각하게 비효율적인게 없죠.

Account-4-Review replied:
또한, 어떤 데이터 존재여부를 확인하고, 존재하지 않으면 예외를 뱉는 코드는 무조건 공통화를 하는게 재사용성을 높여줄겁니다.

Account-4-Review commented on backend/src/main/java/com/ll/hotel/domain/booking/payment/controller/PaymentController.java:
```diff
@@ -22,38 +22,25 @@ public class PaymentController {
     }
 
+    /*
+     * 본래 결제 api와 예약 api는 독립적으로 설계되어 프론트에서 트랜잭션을 수행할 계획이었으나
```
이건 좀 더 이야기 해보도록 할게요.

Account-4-Review commented on backend/src/main/java/com/ll/hotel/domain/hotel/option/cotroller/HotelOptionController.java:
```diff
@@ -44,29 +44,29 @@ public RsData<List<HotelOptionDto>> getAll() {
-        hotelOptionService.modify(hotelOption, details);
+        hotelOptionService.modify(hotelOption, optionRequest);
 
         hotelOptionService.flush();
```
얘는 없어도 될 것 같은데요.

Account-4-Review reviewed (approved):
프로젝트를 보면서 든 생각들을 간단하게 적어봅니다.

- 처음에 비해서 프로젝트의 컨벤션이 어느정도 잡힌 것 같아요. 다만 아직 그렇지 않은 부분도 있었기 때문에, 이후 프로젝트에서는 좀 더 신경 쓰시면 좋을 것 같습니다.
- API 설계 관점에서 어려움을 겪은 듯한 느낌이 드는데요. 리뷰 코멘트에서 남겼듯이 다른 기업의 openapi 스펙을 꼼꼼히 읽어보면 많은 도움이 될 것 같아요.
- 초반 프로젝트라 기술적인 챌린지에 대해선 넣지 않았는데요. 다음 프로젝트부터는 이 부분도 고려해보시면 좋을 것 같아요. (멘토링 시간에 앞으로의 공부 방향성 및 기술적 고찰에 대해 이야기 해볼게요.)

그동안 프로젝트 하느라 고생 많으셨습니다!

LeeKyoWon committed:
Merge pull request #145 from prgrms-be-devcourse/feature-review-front

[FIX][FE] 페이징 에러 최종 해결

LeeKyoWon committed:
Merge pull request #144 from prgrms-be-devcourse/refactor/Room_RoomDetail_font

[REFACTOR] [FE] 객실 이름 폰트 파란색으로 교체

universechoi committed:
[fix] [fe] response 수정

universechoi committed:
[fix][be] response 수정

universechoi committed:
[fix] [be] 실수로 변경된 파일을 이전 상태로 복원

LeeKyoWon committed:
Merge pull request #147 from prgrms-be-devcourse/refactor-admin/businesss

[fix] hotelDetailResponse 수정

LeeKyoWon committed:
Merge pull request #146 from prgrms-be-devcourse/dev

Dev -> Main 병합

oneplast committed:
[FIX] [BE] [FE] 호텔 검색 필터링 수정 및 호텔 목록 출력 시 하단 여백 증가

universechoi committed:
[fix] [fe] 디버깅용 console.log 삭제 및 오류 설정 및 alert 수정

jurio5 committed:
[feat][BE] 즐겨찾기 컨트롤러 테스트 추가

jurio5 committed:
[feat][BE] 즐겨찾기 서비스 테스트 케이스 추가

jurio5 committed:
[fix][BE] 서비스 컨트롤러 TODO 제거

LeeKyoWon committed:
[Test][BE] Review & ReviewComment 테스트코드 추가

LeeKyoWon committed:
[Test][BE] 리뷰 & 답변 테스트 케이스 보충

LeeKyoWon committed:
[Refactor][BE] RsData, ErrorCode 리팩토링 및 Review & Comment 에 적용

LeeKyoWon committed:
[Refactor][BE] ReviewService 가독성을 위해 Review 의 Builder 제거

LeeKyoWon committed:
[Refactor][BE] ReviewController 의 로직들 ReviewService 로 넣기

jurio5 committed:
[feat][BE] OAuth2 통합 테스트 케이스 추가

jurio5 committed:
[feat][BE] JWT 테스트 케이스 추가

LeeKyoWon committed:
Merge pull request #148 from prgrms-be-devcourse/fix/Search_filter

[FIX] [BE] [FE] 호텔 검색 필터링 수정 및 호텔 목록 출력 시 하단 여백 증가

LeeKyoWon committed:
Merge pull request #149 from prgrms-be-devcourse/fix/refactor-admin/business

[fix] [fe] 디버깅용 console.log 삭제 및 오류 설정 및 alert 수정

LeeKyoWon committed:
Merge branch 'dev' of https://github.com/prgrms-be-devcourse/NBE3-4-2-Team01 into feature-review

LeeKyoWon committed:
[Refactor][BE] TestInit 함수명 수정

universechoi committed:
[be][fix] 잘못된 URL 접근 경로 수정

universechoi committed:
[be][fix] PUT 대신 PATCH 메서드 사용하도록 수정

universechoi committed:
[be][test] 사업자 등록 실패 테스트 추가

LeeKyoWon committed:
Merge pull request #151 from prgrms-be-devcourse/feature-review

[Feat][BE] 리뷰 & 답변 테스트 코드 추가

LeeKyoWon committed:
Merge pull request #150 from prgrms-be-devcourse/feature-member

[FEAT] [BE] 즐겨찾기 & JWT & OAuth2 테스트 케이스 추가

universechoi committed:
[be][fix] TransientObjectException 방지를 위해 삭제 로직 제거

universechoi committed:
[be][fix] 하드코딩된 데이터 검증 제거

universechoi committed:
[be][fix] TransientObjectException 방지를 위해 삭제 로직 제거

universechoi committed:
[be][fix] TransientObjectException 방지를 위해 삭제 로직 제거 및 필수 항목 추가

universechoi committed:
[be][fix] 잘못된 URL 접근 경로 수정

universechoi committed:
[be][fix] 잘못된 메소드 이름 수정

universechoi committed:
[be][test]  BusinessValidationServiceTest 클래스 및 유효하지 않은 사업자 테스트 추가

universechoi committed:
[be][test] BusinessControllerTest 추가

oneplast committed:
[TEST] [BE] 호텔, 객실 테스트 수정 및 추가

sete3683 committed:
[TEST][BE] 예약/결제 컨트롤러 테스트 코드 작성

- 통합 컨트롤러 테스트 코드 작성
- InitData 충돌 방지를 위한 profile 수정

LeeKyoWon committed:
Merge pull request #152 from prgrms-be-devcourse/test-admin/business

[TEST][BE] 기존 테스트 수정 및 추가

LeeKyoWon committed:
Merge pull request #153 from prgrms-be-devcourse/test/Hotel_Room

[TEST] [BE] 호텔, 객실 단위 테스트 수정 및 추가

LeeKyoWon committed:
Merge pull request #154 from prgrms-be-devcourse/test/booking

[TEST][BE] 예약/결제 컨트롤러 테스트 코드 작성

LeeKyoWon committed:
Merge pull request #155 from prgrms-be-devcourse/dev

Dev -> Main 병합 : 테스트 코드 작성

LeeKyoWon committed:
Merge branch 'dev' of https://github.com/prgrms-be-devcourse/NBE3-4-2-Team01 into feature-review-refactor

LeeKyoWon committed:
Merge pull request #156 from prgrms-be-devcourse/feature-review-refactor

[Refactor][BE] RsData & ErrorCode 리팩토링 (+ 리뷰 & 답변을 이에 맞게 수정)

LeeKyoWon committed:
[Refactor][FE] 호텔 목록 조회 API 알맞은 클래스로 이동

jurio5 committed:
[refactor][BE] 즐겨찾기 컨트롤러 RsData 리팩토링

universechoi committed:
[be][refactor] RsData 변경에 따라 API 성공 응답 수정

jurio5 committed:
[feat][BE] JWT,OAuth2,즐겨찾기 관련 ErrorCode 추가

jurio5 committed:
[refactor][BE] MemberController RsData & throw 예외 리팩토링

jurio5 committed:
[fix][BE] 리프레쉬 토큰 예외 메시지 변경

jurio5 committed:
[refactor][BE] OAuth2Controller 리팩토링

jurio5 committed:
[refactor][BE] Member Entity 사업자 체크 예외 리팩토링

jurio5 committed:
[refactor][BE] MemberService 예외처리 리팩토링

jurio5 committed:
[refactor][BE] RefreshTokenService 예외처리 리팩토링

jurio5 committed:
[refactor][BE] JwtAuthFilter 예외처리 리팩토링

jurio5 committed:
[refactor][BE] CustomOAuth2FailureHandler 예외처리 리팩토링

jurio5 committed:
[refactor][BE] Jwt 테스트 케이스 예외처리 리팩토링

oneplast committed:
[REFACTOR] [BE] Hotel & Room 도메인 - RsData 및 ServiceException 리팩토링

sete3683 committed:
[REFACTOR][BE] 예약 상태 코드 리팩토링

- 에러 코드 ErrorCode enum으로 관리하도록 변경
- 컨트롤러의 반환 형식 수정
- 예외 처리 형식 수정
- 예약 완료 처리 Post에서 Patch로 수정
- 일부 상태 코드 변경

sete3683 committed:
[REFACTOR][BE] 결제 상태 코드 리팩토링

- 이전 커밋과 유사한 작업

universechoi committed:
[be][refactor] 오류 처리 변경에 따라 글로벌 익셉션 핸들러 및 오류 코드 추가 후 적용

LeeKyoWon committed:
[Rename][BE] ImageConverter 클래스를 Image 패키지로 옮기기

LeeKyoWon committed:
[Refactor][BE] Exception Code 변경(Bad Request -> Not Found)

LeeKyoWon committed:
[Refactor][BE] 백엔드 리팩토링에 맞게 FrontEnd api 코드 수정

LeeKyoWon committed:
Merge pull request #157 from prgrms-be-devcourse/refactor/Hotel_Room_RsData_ServiceException

[REFACTOR] [BE] Hotel & Room 도메인 - RsData 및 ServiceException 리팩토링

LeeKyoWon committed:
Merge branch 'dev' into refactor/booking-be

LeeKyoWon committed:
Merge pull request #158 from prgrms-be-devcourse/refactor/booking-be

[REFACTOR][BE] 예약, 결제 상태 코드 리팩토링

LeeKyoWon committed:
Merge branch 'dev' into feature-member

LeeKyoWon committed:
Merge pull request #159 from prgrms-be-devcourse/feature-member

[REFACTOR] [BE] 즐겨찾기 & OAuth2 & JWT 리팩토링

LeeKyoWon committed:
[Fix][BE] ErrorCode Merge Conflict 해결

universechoi committed:
merge dev into refacotr-admin/business

LeeKyoWon committed:
Merge pull request #160 from prgrms-be-devcourse/feature-review-refactor

[Fix][BE] ErrorCode 관련 Merge Conflict 해결

LeeKyoWon committed:
Merge pull request #161 from prgrms-be-devcourse/refactor-admin/business

[BE][REFACTOR] 추가된 오류코드 및 RsData 적용

LeeKyoWon committed:
[Fix][BE] 합치는 과정에서 발생한 문제들 해결

LeeKyoWon committed:
Merge pull request #162 from prgrms-be-devcourse/feature-review-refactor

[Fix][BE] 합치는 과정에서 발생한 문제들 해결

jurio5 committed:
[fix][BE] 즐겨찾기 컨트롤러 Validation 문제 해결

jurio5 committed:
[fix][BE] 즐겨찾기 컨트롤러 테스트 케이스 오류 해결

jurio5 committed:
[fix][BE] 즐겨찾기 테스트 예외 메시지 오류 해결

jurio5 committed:
[fix][BE] Jwt 테스트 케이스 상태 코드 관련 문제 해결

LeeKyoWon committed:
Merge pull request #164 from prgrms-be-devcourse/feature-member

[FIX] [BE] 테스트 케이스 문제 해결

LeeKyoWon committed:
[Refactor][BE] 사업가 -> 사업자로 통일

universechoi committed:
[be][fix] 테스트 오류 수정

LeeKyoWon committed:
Merge branch 'dev' of https://github.com/prgrms-be-devcourse/NBE3-4-2-Team01 into feature-review-refactor

LeeKyoWon committed:
Merge pull request #165 from prgrms-be-devcourse/fix-admin/business

[FIX][BE] 테스트 오류 수정

LeeKyoWon committed:
Merge pull request #166 from prgrms-be-devcourse/feature-review-refactor

[TEST][BE] 테스트 코드 정상 동작 버전

LeeKyoWon committed:
[Fix][FE] 백엔드 리팩토링 이후 발생한 Api 문제 해결

LeeKyoWon committed:
[Fix][BE] Empty 클래스, 이에 대한 의존성 제거

LeeKyoWon committed:
Merge pull request #167 from prgrms-be-devcourse/feature-review-refactor

[Fix][BE][FE] 백엔드 리팩토링 이후 발생한 Api 파일들 문제 해결

jurio5 committed:
[fix][BE] 즐겨찾기 컨트롤러 테스트 케이스 상태 코드 문제 해결

LeeKyoWon committed:
Merge pull request #168 from prgrms-be-devcourse/feature-member

[FIX] [BE] 즐겨찾기 컨트롤러 테스트 케이스 상태 코드 문제 해결

sete3683 committed:
[FIX][FE] API 호출 메서드 수정

- 완료 처리 POST에서 PATCH로 변경한 부분 반영

LeeKyoWon committed:
Merge pull request #169 from prgrms-be-devcourse/feature/booking-fe

[FIX][FE] api 변경 사항 반영

LeeKyoWon committed:
[Refactor][BE]application.yml AWS S3 관련 설정 Test/Dev 분리

oneplast committed:
[FEAT] [BE] 로깅 설정 및 AOP 적용

jurio5 committed:
[feat][BE] 간소화 호텔DTO 추가

jurio5 committed:
[refactor][BE] 즐겨찾기 목록 조회 구조 변경
DTO로 프론트에 넘어가는 정보가 많아 프론트 쪽에서 그 정보를 다 받지 못하고 오버플로우가 생겨 즐겨찾기가 1개 이상 출력이 되지 않던 부분 해결

jurio5 committed:
Merge branch 'dev' of https://github.com/prgrms-be-devcourse/NBE3-4-2-team01 into feature-member

jurio5 committed:
[refactor][FE] 프론트 HotelDto 데이터 값 리팩토링

jurio5 committed:
[fix][FE] 회원가입 fetch response.json() 파싱 및 예외 처리 추가

jurio5 committed:
[fix][FE] 회원가입 상태코드 파싱 문제 해결

universechoi committed:
[fe][refactor] 백엔드 응답 형식 변경에 따른 fetchAPI 로직 수정

sete3683 committed:
[REFACTOR][FE] 예외 메세지 파싱 수정

- 예외 메세지의 파싱 방식 수정

oneplast committed:
[REFACTOR] [FE] 호텔, 객실 프론트 응답 및 예외 메시지 처리

LeeKyoWon committed:
Merge branch 'dev' of https://github.com/prgrms-be-devcourse/NBE3-4-2-Team01 into feature-review-refactor

LeeKyoWon committed:
Merge pull request #171 from prgrms-be-devcourse/feat/Loging_By_Aspect

[FEAT] [BE] 로깅 설정 및 AOP 적용

LeeKyoWon committed:
Merge pull request #174 from prgrms-be-devcourse/refactor-admin/business

[REFACTOR][FE] 백엔드 응답 구조 변경에 따라 api 로직 수정

LeeKyoWon committed:
Merge pull request #170 from prgrms-be-devcourse/feature-review-refactor

[Refactor][BE]application.yml AWS S3 관련 설정 Test/Dev 분리

LeeKyoWon committed:
Merge pull request #175 from prgrms-be-devcourse/feature/booking-fe

[REFACTOR][FE] 예외 메세지 파싱 수정

LeeKyoWon committed:
Merge branch 'dev' of https://github.com/prgrms-be-devcourse/NBE3-4-2-Team01 into feature-review-refactor

LeeKyoWon committed:
Merge pull request #176 from prgrms-be-devcourse/refactor/Response_Message

[REFACTOR] [FE] 호텔, 객실 프론트 응답 및 예외 메시지 처리

oneplast committed:
[REFACTOR] [BE] 코드 단순화

universechoi committed:
[be][refactor] 스웨거 문서를 위한 tag, opertation 작성 및 사용하지 않는 메서드 삭제

jurio5 committed:
[feat][FE] FavoriteHotelDto 마운트 추가

jurio5 committed:
[feat][FE] FavoriteHotelDto 타입 변경

jurio5 committed:
[delete][FE] 기존 HotelDto.ts 제거

jurio5 committed:
[feat][FE] FavoriteHotelDto.ts 추가

jurio5 committed:
[feat][BE] FavoriteDTO 추가

jurio5 committed:
[rename][BE] FavoriteDTO Rename

jurio5 committed:
[refactor][BE] 즐겨찾기 컨트롤러 getFavorites 반환 타입 변경

jurio5 committed:
[fix][BE] MemberController 중복 예외 수정

jurio5 committed:
[fix][BE] MemberEntity 중복 예외 수정

jurio5 committed:
[fix][BE] MemberService 중복 예외 수정

jurio5 committed:
[fix][BE] RefreshTokenService 중복 예외 수정

jurio5 committed:
[fix][BE] JwtAuthFilter 중복 예외 수정

jurio5 committed:
[fix][BE] 즐겨찾기 테스트 호텔 타입 변경

jurio5 committed:
Merge branch 'dev' of https://github.com/prgrms-be-devcourse/NBE3-4-2-team01 into feature-member

sete3683 committed:
[FEAT][BE] Swagger 명세 작성

- 예약, 결제 api 관련 swagger 명세 작성

LeeKyoWon committed:
[RENAME][FE] 패키지 구조 수정 & 패키지명 소문자로 통일 & 불필요한 import 제거

jurio5 committed:
[delete][BE] HotelDTO del

LeeKyoWon committed:
Merge pull request #179 from prgrms-be-devcourse/feature-member-front

[FEAT] [FE] 즐겨찾기 DTO 추가 및 회원가입 파싱 문제 해결

jurio5 committed:
[refactor][BE] 임포트 정리

jurio5 committed:
Merge branch 'dev' of https://github.com/prgrms-be-devcourse/NBE3-4-2-team01 into feature-member

LeeKyoWon committed:
[Fix][FE] FavoriteDto 관련 Merge Conflict 해결

LeeKyoWon committed:
Merge pull request #183 from prgrms-be-devcourse/feature-review-refactor

[RENAME][FE] 패키지 구조 수정 & 패키지명 소문자로 통일 & 불필요한 import 제거

LeeKyoWon committed:
Merge pull request #182 from prgrms-be-devcourse/feature-member

[REFACTOR] [BE] 즐겨찾기 DTO 데이터 구조 변경 및 중복 예외 처리

LeeKyoWon committed:
Merge pull request #177 from prgrms-be-devcourse/refactor-admin/business

[REFACTOR][BE]스웨거 문서를 위한 tag, opertation 작성 및 사용하지 않는 메서드 삭제

LeeKyoWon committed:
Merge pull request #181 from prgrms-be-devcourse/feature/booking-be

[FEAT][BE] 예약, 결제 파트 swagger 명세 작성

LeeKyoWon committed:
Merge branch 'dev' into refactor/aspect

LeeKyoWon committed:
Merge pull request #178 from prgrms-be-devcourse/refactor/aspect

[REFACTOR] [BE] 코드 단순화

LeeKyoWon committed:
[Remove][BE] 리뷰 답변 단건 조회 코드 제거

jurio5 committed:
[delete][BE] 불필요한 코드 제거

LeeKyoWon committed:
[Refactor][BE]  throw Exception 코드 개선

jurio5 committed:
[delete][BE] 불필요한 코드 제거

jurio5 committed:
Merge branch 'dev' of https://github.com/prgrms-be-devcourse/NBE3-4-2-team01 into feature-member

LeeKyoWon committed:
Merge pull request #184 from prgrms-be-devcourse/feature-review-refactor

[Refactor][BE] 리뷰 답변 단건 조회 제거 + throw Exception 구조 개선

jurio5 committed:
[feat][BE] swagger 코드 추가

jurio5 committed:
[feat][BE] swagger 코드 추가

oneplast committed:
[DOCUMENT] [BE] Hotel, Room 도메인 Swagger 작업

jurio5 committed:
Merge branch 'dev' of https://github.com/prgrms-be-devcourse/NBE3-4-2-team01 into feature-member

LeeKyoWon committed:
Merge pull request #185 from prgrms-be-devcourse/feature-member

[FEAT] [BE] 즐겨찾기 , 회원 swagger 코드 추가

jurio5 committed:
[delete][BE] 불필요한 로그 제거

jurio5 committed:
[delete][BE] 불필요한 로그 제거

jurio5 committed:
[delete][BE] 불필요한 로그 제거

LeeKyoWon committed:
Merge pull request #186 from prgrms-be-devcourse/refactor/Hotel_Room_Swagger

[DOCUMENT] [BE] Hotel, Room 도메인 Swagger 작업

jurio5 committed:
[delete][BE] 불필요한 로그 제거

jurio5 committed:
[delete][BE] 불필요한 로그 제거

jurio5 committed:
Merge branch 'dev' of https://github.com/prgrms-be-devcourse/NBE3-4-2-team01 into feature-member

LeeKyoWon committed:
[Rename][BE] 패키지명 소문자, 알맞게 변경

LeeKyoWon committed:
Update PULL_REQUEST_TEMPLATE.md

LeeKyoWon committed:
Merge pull request #187 from prgrms-be-devcourse/feature-review-refactor

[Rename][BE] 패키지명 소문자, 알맞게 변경

jurio5 committed:
[fix][BE] merge fix

jurio5 committed:
[fix][BE] RefreshTokenService 임포트 수정

oneplast committed:
[REFACTOR] [BE] 테스트코드 실행 시에는 파일에 로그가 남지 않도록 수정

universechoi committed:
[be][fix] 잘못 삭제된 함수 복구

universechoi committed:
[be][test] 삭제된 함수에 대한 테스트 삭제

LeeKyoWon committed:
Merge pull request #188 from prgrms-be-devcourse/refactor/log

[REFACTOR] [BE] 테스트코드 실행 시에는 파일에 로그가 남지 않도록 수정

LeeKyoWon committed:
Merge pull request #189 from prgrms-be-devcourse/fix-admin/business

[FIX][BE] 잘못 삭제된 함수 복구

LeeKyoWon committed:
Merge pull request #190 from prgrms-be-devcourse/feature-member

[FIX] [BE] 불필요한 로그 제거

LeeKyoWon committed:
Merge branch 'main' into dev

LeeKyoWon committed:
[Feat][BE] 객실 옵션 수정 Swagger 추가

LeeKyoWon committed:
Merge pull request #193 from prgrms-be-devcourse/feature-review-refactor

[Feat][BE] 객실 옵션 수정 Swagger 추가

LeeKyoWon committed:
[Refactor][FE] 패키지명 소문자로 통일

LeeKyoWon committed:
Git 대소문자 구분 안하는 문제 해결

LeeKyoWon committed:
Merge pull request #194 from prgrms-be-devcourse/feature-review-refactor

[Refactor][FE] 깃 패지키 소문자로 변경, prettier 적용

LeeKyoWon committed:
[Rename][FE] Pagination -> pagination 으로 수정

LeeKyoWon committed:
Merge pull request #196 from prgrms-be-devcourse/feature-review-refactor

[Rename][FE] Pagination -> pagination 으로 수정

oneplast committed:
[REFACTOR] [BE] AOP 로그 출력 형식 변경, 호출되는 서비스도 로그파일에 추가

LeeKyoWon committed:
Merge pull request #198 from prgrms-be-devcourse/refactor/log

[REFACTOR] [BE] AOP 로그 출력 형식 변경, 호출되는 서비스도 로그파일에 추가

oneplast committed:
[FIX] [FE] Param 제대로 못받는 오류 수정

LeeKyoWon committed:
Merge pull request #200 from prgrms-be-devcourse/fix/Input_Param

[FIX] [FE] Param 제대로 못 받는 오류 수정

oneplast committed:
[REFACTOR] [BE] 로그 관련 설정 변경

LeeKyoWon committed:
Merge pull request #202 from prgrms-be-devcourse/refactor/Log_Config

[REFACTOR] [BE] 로그 관련 설정 변경

oneplast committed:
[REFACTOR] [BE] 중복 에러 로그 저장 문제 수정

LeeKyoWon committed:
Merge pull request #203 from prgrms-be-devcourse/refactor/Log_Config

[REFACTOR] [BE] 중복 에러 로그 저장 문제 수정

oneplast committed:
[REFACTOR] [FE] checkoutDate -> checkOutDate로 변경

LeeKyoWon committed:
Merge pull request #204 from prgrms-be-devcourse/refactor/Invalid_paramName

[REFACTOR] [FE] checkoutDate -> checkOutDate로 변경

oneplast committed:
[REFACTOR] [BE] Hotel, Room 리팩토링

LeeKyoWon committed:
Merge pull request #205 from prgrms-be-devcourse/refactor/Hotel_Room

[REFACTOR] [BE] Hotel, Room 리팩토링

oneplast committed:
[REFACTOR] [FE] 수정, 삭제 시 페이지 전환 처리 수정 및 UI 개선

LeeKyoWon committed:
Merge pull request #206 from prgrms-be-devcourse/refactor/fe/Hotel_Room

[REFACTOR] [FE] 수정, 삭제 시 페이지 전환 처리 수정 및 UI 개선
